wizard/WizardDaemonSettings.qml

```diff
@@ -92,6 +92,35 @@ ColumnLayout {
                     blockchainFolder.focus = true;
                 }
             }
+            onTextUpdated: {
+                existingDbWarning.visible = daemonManager.validateDataDir(persistentSettings.blockchainDataDir).lmdbExists
+            }
+        }
+
+        RowLayout {
+            id: pruningOptionRow
+            MoneroComponents.CheckBox {
+                id: pruneBlockchainCheckBox
+                checked: persistentSettings.pruneBlockchain
+                onClicked: {
+                    if (daemonManager.validateDataDir(persistentSettings.blockchainDataDir).lmdbExists) {
```

Comment:
As far as I can see persistentSettings.blockchainDataDir is only set if a custom blockchain data dir is used, so if someone has a blockchain in the default path this does not work.

Reply:
gotcha